CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [v2.11.1](http://github.com/ndustrialio/contxt-sdk-js/tree/v2.11.1) (2020-02-27)
+
+**Updated**
+
+- Updated Asset Metric Value documentation to support `is_estimated` column
```

Comment:
Should this be `is_estimated` or `isEstimated`? While I understand the language here. I feel like what you actually pass to the create or update method is `isEstimated` and not `is_estimated`?